lib/acts_as_taggable_on_mongoid/default_parser.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module ActsAsTaggableOnMongoid
```

Comment:
Copied from here:
https://github.com/mbleigh/acts-as-taggable-on/blob/master/lib/acts_as_taggable_on/default_parser.rb

Added to_s, and removed creation of a TagList here.